drivers/mdio/mdio_gpio.c

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
Squash all commits that fix what the review points.

Reply:
Probably outdated?